core/src/epicli/data/common/defaults/configuration/shared-config.yml

```diff
@@ -7,5 +7,8 @@ specification:
   download_directory: /tmp # directory where files and images will be stored just before installing/loading
   vault_location: '' # if empty "BUILD DIRECTORY/vault" will be used
   vault_tmp_file_location: SET_BY_AUTOMATION
+  supported_architecture:
```

Comment:
I don't think this is a right place for this property for two reasons:
1) supported components are not configurable, they are defined by us (dev team) not by users
2) this property is used only by preflight role so it's better to be local
My proposal is to move it to: `data/common/ansible/playbooks/roles/preflight/defaults/main.yml`